docs/source/examples/Image_reconstruction.rst

```diff
@@ -0,0 +1,320 @@
+.. role:: hidden
+    :class: hidden-section
+
+
+.. _intro:
+
+Image reconstruction using Poutyne
+***********************************
+
+.. note::
+
+    - See the notebook `here <https://github.com/GRAAL-Research/poutyne/blob/master/examples/Image_reconstruction.ipynb>`_
+    - Run in `Google Colab <https://colab.research.google.com/github/GRAAL-Research/poutyne/blob/master/examples/Image_reconstruction.ipynb>`_
+
+In this example, we train a simple convolutional autoencoder (Conv-AE) on the MNIST dataset to learn image reconstruction. The Conv-AE is composed of two parts: an encoder and a decoder. The encoder encodes the input images to extract compact image features. The decoder, on the other hand, decodes the extracted features, to reconstruct the input images.
```

Comment:
would you please make me more clear about this comment

**It seems that the build of the doc has warnings in it and that you did not add your page in the menu. Can you fix the warnings and include your page in the menu?**

Do you mean I add the link of my own git and collab?

**Finally, could you create a directory for your images with the name image_reconstruction instead of putting at the root of the img directory? I know we don't do that for the others but the directory will have too many images in it soon if we don't start now.**

Should I create this directory in the dev branch, inside the _static directory? and set the path to this directory? Do I have permission (in GitHub) to create the directory?

Reply:
> Do you mean I add the link of my own git and collab?

In the docs directory, there is a script called `rebuild_html_doc.sh` which you can call via the command line in this way: `./rebuild_html_doc.sh`. This script builds the documentation (i.e. the website) in a subdirectory called `_build/html/`. You can then visualize the website by opening the `index.html` or by starting an HTTP server with Python: `python -m http.server --directory _build/html/` and go to the URL `http://localhost:8000/` afterwards. When I talk about menu, I'm talking about the menu on the website.

> Should I create this directory in the dev branch, inside the _static directory? and set the path to this directory? Do I have permission (in GitHub) to create the directory?

A directory in `_static/img/`. You can it inside the current branch. You don't need any permission since you are currently on your own Github repo.